route/route-free-trial-request.js

```diff
@@ -50,9 +51,10 @@ module.exports = router => {
 
```

Comment:
We should also validate that ```req.body.instument``` is uppercase, or force it with ```req.body.instrument.toUpperCase()```, before saving Student to the DB.